src/components/standalone/firewall/rules/SourceOrDestinationRuleColumn.vue

```diff
@@ -160,20 +160,17 @@ const zone = computed(() => {
         </p>
       </template>
     </template>
-    <template v-else>
+    <template v-else-if="zone === '*'">
```

Comment:
Since zone is always a string

```suggestion
    <template v-else-if="zone == '*'">
```